tests/basics/int1.py

```diff
@@ -79,6 +81,8 @@ def test(value, base):
 test('0xg', 16)
 test('1 1', 16)
 test('123', 37)
+test('01', 0)
+test('000 ', 0)
```

Comment:
This test is supposed to pass, so I suggest it goes above as simply `print(int('000 ', 0))`.  Also maybe test with non-zero bases as well:
```py
print(int('000 ', 10))
print(int('000 ', 16))
```